src/reactapp/src/pages/EvaluationForm.js

```diff
@@ -4,119 +4,255 @@
 
 import {
-  Box,
-  Container,
-  Stack,
-  Text,
-  VStack,
-  Button,
+    Box,
+    Button,
+    ButtonGroup,
+    Container,
+    FormControl,
+    FormLabel,
+    HStack,
   Heading,
-  StackDivider,
-  useColorModeValue,
-  HStack,
-  FormControl,
-  FormLabel,
-  ButtonGroup,
-  Table,
-  Textarea,
-  Thead,
-  Tbody,
-  Tr,
-  Th,
-  Td,
+    Skeleton,
+    Stack,
+    StackDivider,
+    Table,
+    Tbody,
+    Td,
+    Text,
+    Textarea,
+    Th,
+    Thead,
+    Tr,
+    VStack,
+    useColorModeValue,
+    useToast,
 } from "@chakra-ui/react";
-import "../App.css";
 import React, { useEffect, useState } from "react";
-import { useLocation } from "react-router-dom";
 import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { useLocation, useNavigate, useParams } from "react-router-dom";
 import Select from 'react-select';
+import "../App.css";
 import { get, post } from "../components/FetchHelpers";
-import { getLoggedInUserName, getLoggedInUserRole, getLoggedInUserId } from "../components/TokenHelpers";
-import "react-datepicker/dist/react-datepicker.css";
+import { AlertMessage } from "../components/AlertMessage";
+import { getLoggedInUserId, getLoggedInUserName, getLoggedInUserRole } from "../components/TokenHelpers";
+import { AlertMessageDialog } from "../components/AlertMessageDialog";
 
 export default function EvaluationForm() {
+  const [isEvaluationLoaded, setIsEvaluationLoaded] = useState(false);
+  const [componentsDataLoaded, setComponentsDataLoaded] = useState(false);
   const [loggedInUser, setLoggedInUser] = useState({ "name": "", "role": null });
+  const [evaluationDataLoaded, setEvaluationDataLoaded] = useState({});
   const [selectedCandidate, setSelectedCandidate] = useState({});
   const [selectedEvaluation, setSelectedEvaluation] = useState({});
   const [ratingLevelOptions, setRatingLevelOptions] = useState([]);
   const [evaluationMetadata, setEvaluationMetadata] = useState({});
-
+  const [currentEvaluator, setCurrentEvaluator] = useState({});
   const [evaluationDate, setEvaluationDate] = useState(new Date());
   const [evaluationEndTime, setEvaluationEndTime] = useState(null);
   const [elementRatings, setElementRatings] = useState([]);
   const [objectiveNotes, setObjectiveNotes] = useState([]);
+  const [performanceEvaluationData, setPerformanceEvaluationData] = useState(null);
+  const [userHasAccessToEvaluation, setUserHasAccessToEvaluation] = useState(true);
+  const [evaluationLoaded, setEvaluationLoaded] = useState(true);
+  const [alertMessageText, setAlertMessageText] = useState(true);
+  const { id } = useParams();
+  const location = useLocation();
+  const navigate = useNavigate();
+  const toast = useToast();  
+  const borderColor = useColorModeValue("gray.200", "gray.600"); 
+  // This will be replaced with data from the ODS/API once EPPETA-38 is complete
+  const ratingLevels = [
```

Comment:
@jleiva-gap EPPETA-38 has been completed and merged into main. We're no longer using this hard-coded rating levels object. I noticed the that https://api.ed-fi.org/v6.1/ was missing the rating levels on PerformanceEvaluation again so I've re-added them. Please update this to pull rating level information from the evaluation like in the main branch

Reply:
I updated the PR to remove those hard-coded levels.